filament/src/details/RenderTarget.cpp

```diff
@@ -105,28 +116,40 @@ RenderTarget* RenderTarget::Builder::build(Engine& engine) {
     uint32_t maxWidth = 0;
     uint32_t minHeight = std::numeric_limits<uint32_t>::max();
     uint32_t maxHeight = 0;
-    uint32_t minDepth = std::numeric_limits<uint32_t>::max();
-    uint32_t maxDepth = 0;
+    uint32_t minLayerCount = std::numeric_limits<uint32_t>::max();
+    uint32_t maxLayerCount = 0;
     for (auto const& attachment : mImpl->mAttachments) {
         if (attachment.texture) {
             const uint32_t w = attachment.texture->getWidth(attachment.mipLevel);
             const uint32_t h = attachment.texture->getHeight(attachment.mipLevel);
             const uint32_t d = attachment.texture->getDepth(attachment.mipLevel);
+            const uint32_t l = attachment.layerCount;
```

Comment:
technically speaking, shouldn't we always have at least one layer?  Should this be `layer` here instead of `layerCount`?

Reply:
sorry I disabled auto-merge while I still have this one question.

Reply:
layerCount here means the number of layers used for multiview. This variable indicates whether we use multiview . If this value is 0, then it means we don't use multiview for this attachment.

Reply:
why not just make it the same as `mLayerCount` in `BuilderDetails` so that `> 1` means that multiview is enabled?  So we don't have two different interpretations of the same variable name.

Reply:
That was my original implementation. But the new interface `.multiview(attachpoint, layerCount, startingLayer)` requires each attachment point to have a layerCount.